db/migrate/20200513092446_change_config_chart_class.rb

```diff
@@ -0,0 +1,10 @@
+class ChangeConfigChartClass < ActiveRecord::Migration[6.0]
+  def up
```

Comment:
Since `down` migration would not be as useful in this case, can we use `change` ?

Reply:
Since this isn't automatically reversible code (like e.g. `add_table`), I think if it was in `change` it would also run on down migration which isn't needed.

Reply:
got it. The PR looks great!